invenio_opendata/base/recordext/fields/opendata.cfg

```diff
@@ -305,6 +321,14 @@ reprocessing_date:
     producer:
         json_for_marc(), {"960__c": ""}
 
+# 964_0c
+
+run_number:
```

Comment:
"Data-taking period" or "Run period". I've seen also "era" but it sounds a bit too epic to me...